Dockerfile

```diff
@@ -52,8 +52,8 @@ RUN ARCH=$(uname -m) && \
       libxslt-devel \
       make \
       openssl-devel \
-      python38-devel \
-      python38-pip \
+      python3-devel \
+      python3-pip \
```

Comment:
Is it possible to do python39-devel and python39-pip?  Reason is that while it may be the system python, moving forward we're going to want to go to 3.10, 3.11, etc, and it's so much easier if we are explicit on the version.

Reply:
No, python 3.9 RPMs are python3-*.  There is no python3.10.  Python 3.11 is `python-3.11` or `python-311`, but not all of the dependencies that we need are there yet.